assets/src/scripts/monitoring-location/components/daily-value-hydrograph/graph-brush.js

```diff
@@ -73,14 +93,54 @@ export const drawGraphBrush = function(container, store) {
             const group = svg.append('g').attr('class', 'brush')
                 .attr('transform', `translate(${layout.margin.left},${layout.margin.top})`);
 
+            graphBrush.handleSize([1]); // make default handle 1px wide
             graphBrush.extent([[0, 0], [layout.width - layout.margin.right, layout.height - layout.margin.bottom - layout.margin.top]]);
 
+            /* Draws the custom brush handle using an SVG path. The path is drawn twice, once for the handle
+            * on the left hand side, which in d3 brush terms is referred to as 'east' (data type 'e'), and then
+            * inverted for the right hand custom handle. Here 'east' will be a value of either 1 or 0 (in effect, making
+            * it a boolean value of 'east' or 'not east' */
+            const brushResizePath = function(d) {
+                let east = d.type === 'e' ? 1:0,
+                    x = east ? 1 : -1,
```

Comment:
I need to see if there is a linting rule for this. IOW's convention is to have each variable declared separately. Also all three of these variables are constants therefore should use 'const' rather than 'let'.

Reply:
Ah, now that you mention it, I seem to recall a distant memory of that rule. :) Fixed.